src/RepliesAndForwards/RepliesAndForwards.js

```diff
@@ -18,7 +18,6 @@ const RepliesAndForwards = (props) => {
   } = props;
   const styleObj = {
     color: 'black',
-    fontFamily: '"Segoe UI", Helvetica, Arial, sans-serif',
```

Comment:
We probably want to keep all of the explicit fonts given email clients.

Drop segoe though, keep helvetica, arial, sans